### ISSUE TITLE
Meteor 1.2: no method 'HmacSHA256' error

### ISSUE THREAD

**Issue by rbaindourov** [p-j/meteor-crypto-hmac#1]
Hi There,

I recently created a small NodeJS [example](https://github.com/rbaindourov/paysimple-node) of how to connect to PaySimple.com API, because my real goal was to have  baseline to do it in meteor.

Your package seems like the main way to go for hmac sha256 crypto. But I am getting this strange error even though I pretty much followed the instructions.

```
Exception in setInterval callback: TypeError: Object [object Object] has no method 'HmacSHA256'
at server/init.js:8:29
at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
at packages/meteor/timers.js:6:1
at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1)
```

Here is the snippet. I'm just trying to create the headers and start polling their API, but I get a no method error.

```
  Meteor.startup( () => {
      console.log( 'startup');

      Meteor.setInterval( ()=>{

        let iso =  new Date().toISOString()
        let hmac = CryptoJS.HmacSHA256( iso, Meteor.settings.secret ).toString();

        console.log( 'polling paysimple', Meteor.settings, hmac);

        request.get(Meteor.settings.url, (err, response, body)=>{

          console.log( body );

        });

      }, 1000)
  });
```

Thanks in Advance!

**Comment by rbaindourov** [comment]
Its come to my attention that this packages existence probably serves more to confuse then to help.
I ended up using NodeJS built in crypto package.

At first I made the mistake of trying to use meteorhacks:npm and installing crypto from npm, but this package too probably should be named differently, since its collides with this built in NodeJS package.

``` javacript

  let crypto = Npm.require('crypto')

  Meteor.startup( () => {
      console.log( 'startup');

      Meteor.setInterval( ()=>{

        let iso =  new Date().toISOString()
        let hmac = crypto.createHmac('sha256', Meteor.settings.paysimple.secret).update(new Buffer( iso ).toString() ).digest('base64')

        let headers = {
          'Authorization': `PSSERVER AccessId = ${Meteor.settings.paysimple.username}; Timestamp = ${iso}; Signature = ${hmac}`
        }

        request.get({url:Meteor.settings.paysimple.url, strictSSL:false, headers:headers}, (err, response, body)=>{
          console.log( err, response, body );
        });

      }, 5000)

  });
```

**Comment by p-j** [comment]
Well, without more information on the setup of you Meteor project it is hard to help.
This kind of error has been reported before and it was usually a misconfiguration.

Anyway, if you've found a working solution, that's great.

The main interest of this package over the npm's one is that its been built from the ground up to work in the browser as well, and that I've kept the CryptoJS library as modular as possible since you don't want to ship useless code to the browser.

Cheers.

**Comment by rbaindourov** [comment]
Thank you for your reply,

I am new to this ecosystem, so I apologize at my stumbling around, and perpetual confusion.

The setup is a clean Meteor 1.2 project with React and RequestJS. 
Thats pretty much it in terms of add ons, for now.
The project resides [here](https://github.com/rbaindourov/paysimple-meteor) on github.

Its still messy, I know, but I've committed what I have already, and unfortunately it doesn't demonstrate the error any longer, since I've moved past that. 

Thanks Again!
